src/qt/receivecoinsdialog.cpp

```diff
@@ -266,9 +266,18 @@ void ReceiveCoinsDialog::copyColumnToClipboard(int column)
 // context menu
 void ReceiveCoinsDialog::showMenu(const QPoint &point)
 {
-    if (!selectedRow().isValid()) {
+    const QModelIndex sel = selectedRow();
+    if (!sel.isValid()) {
         return;
     }
+
+    // disable context menu actions when appropriate
+    const RecentRequestsTableModel* const submodel = model->getRecentRequestsTableModel();
+    const RecentRequestEntry& req = submodel->entry(sel.row());
+    copyLabelAction->setDisabled(req.recipient.label.isEmpty());
+    copyMessageAction->setDisabled(req.recipient.message.isEmpty());
+    copyAmountAction->setDisabled(req.recipient.amount == 0);
```

Comment:
```suggestion
    const auto& r = model->getRecentRequestsTableModel()->entry(sel.row()).recipient;
    copyLabelAction->setDisabled(r.label.isEmpty());
    copyMessageAction->setDisabled(r.message.isEmpty());
    copyAmountAction->setDisabled(r.amount == 0);
```

Reply:
Since 213 was merged with the style used in this PR, I'll close this for consistency